submissions/LGeorge007/DOM-practice/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <input type="checkbox" id="nav-toggle" hidden>
```

Comment:
No need to use `checkbox` for open/close menu in this task. you can use a simple button with toggling classes.